docs/_articles/en/v3.x/faq.md

```diff
@@ -110,44 +110,6 @@ Graph Engine builds up the context of the source code in its entirety before it
 
 To learn how, read about [Engine Directives](./en/v3.x/salesforce-graph-engine/working-with-sfge/#add-engine-directives).
 
-## Questions about Interpreting ApexFlsViolationRule results
```

Comment:
Can we keep the title? It's still relevant to the next question in the FAQ.

Reply:
@rmohan20 Sure thing: I've made this edit.